cmd/ebrelayer/relayer/ethereum.go

```diff
@@ -364,21 +359,21 @@ func (sub EthereumSub) Replay(fromBlock int64, toBlock int64, cosmosFromBlock in
 	bridgeClaims := sub.getAllClaims(cosmosFromBlock, cosmosToBlock)
 	log.Printf("found out %d bridgeClaims\n", len(bridgeClaims))
 
-	client, err := SetupRPCEthClient(sub.EthProvider)
+	c, err := SetupRPCEthClient(sub.EthProvider)
```

Comment:
This variable has to be renamed. It is so non descriptive. ethRPCClient would work